### ISSUE TITLE
Move /statuses options to /presence command

### ISSUE THREAD

**Issue by boothj5** [boothj5/profanity#815]
Currently to show/hide presence updates in the console/chat/room windows, the `/statuses` command must be used.

On a few occasions users have asked how to change these settings as `/statuses` is probably not the first thing people think of.

Move setting of these options to the `/presence` command, as it's probably a bit more obvious.

The current `/presence` command only sets whether a contact's presence is shown in the titlebar:

```
/presence on|off
```

Change to:

```
/presence titlebar on|off
```

And add the following (from `/statuses`):

```
/presence console all|online|none
/presence chat all|online|none
/presence room all|online|none
```

**Comment by boothj5** [comment]
Updated in master.

**Comment by andywgrant** [comment]
This update makes sense, but the user guide (http://www.profanity.im/customui.html) should be updated to reflect this. Couldn't figure out why `/statuses` wasn't a valid command until finding this closed issue.

**Comment by boothj5** [comment]
I've removed the page above.  It's an old page and there are no links to it in the site, but I guess search engines may still be indexing it.

The command reference (http://www.profanity.im/reference.html) is generated, but the custom ui page above required updating manually when commands change etc so was a bit of an overhead.  Also the inline help now allows listing commands that may be used to customise the user interface:

```
/help commands ui
```

After searching the site source, there was also a broken link to the non existent `/statuses` command in http://www.profanity.im/themes.html which I've just fixed.

Thanks.
